edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -254,6 +255,21 @@ static string GetStoragePath(IConfiguration configuration)
             return storagePath;
         }
 
+        static string GetConfigurationSourceFilePath(IConfiguration configuration, ILogger logger)
+        {
+            string configFileName = "config.json";
+            string configSourceBindPath = configuration.GetValue<string>("ConfigSourceBindPath");
+
+            if (string.IsNullOrWhiteSpace(configSourceBindPath) || !Directory.Exists(configSourceBindPath))
```

Comment:
So you expect this to be a folder? Why not let user specify the entire path - folder + filename?

Reply:
Changed it to a file path now